src/System.Device.Gpio/System/Device/Pwm/PwmChannel.cs

```diff
@@ -7,8 +7,15 @@ namespace System.Device.Pwm
     /// <summary>
     /// Represents a single PWM channel.
     /// </summary>
-    public abstract partial class PwmChannel : IDisposable
+    public abstract class PwmChannel : IDisposable
     {
+        public Board Board { get; }
+
+        protected PwmChannel(Board board)
```

Comment:
This would be removing the parameterless constructor on PwmChannel which would be a breaking change. If we go with these changes we would have to add that constructor back.